test/Microsoft.AspNetCore.Mvc.Core.Test/FileContentResultTest.cs

```diff
@@ -134,18 +134,34 @@ public void Constructor_SetsLastModifiedAndEtag()
             httpResponse.Body.Seek(0, SeekOrigin.Begin);
             var streamReader = new StreamReader(httpResponse.Body);
             var body = streamReader.ReadToEndAsync().Result;
-            Assert.Equal(StatusCodes.Status206PartialContent, httpResponse.StatusCode);
-            Assert.Equal("bytes", httpResponse.Headers[HeaderNames.AcceptRanges]);
-            var contentRange = new ContentRangeHeaderValue(start.Value, end.Value, byteArray.Length);
-            Assert.Equal(contentRange.ToString(), httpResponse.Headers[HeaderNames.ContentRange]);
             Assert.Equal(lastModified.ToString("R"), httpResponse.Headers[HeaderNames.LastModified]);
             Assert.Equal(entityTag.ToString(), httpResponse.Headers[HeaderNames.ETag]);
-            Assert.Equal(contentLength, httpResponse.ContentLength);
-            Assert.Equal(expectedString, body);
+
+#if NET451
```

Comment:
This should be NET461, but the fact that the else worked likely means you don't need it at all. I assume this was copied from something outdated?

Reply:
Yeah this might not be needed at all if the switch APIs are available in netstandard20. The sample it's copied from is designed for things back to .NET 4.5.1.